source/features/checks-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import {$} from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import features from '../feature-manager.js';
+
+function onButtonClick({delegateTarget: delegate, target}: DelegateEvent): void {
+	// Only close if clicking outside of modal
+	if (delegate === target) {
+		$('[class^="Dialog__DialogCloseButton-"], [class*=" Dialog__DialogCloseButton-"]', delegate)?.click();
+	}
+}
+
+function init(signal: AbortSignal): void {
+	delegate('#__primerPortalRoot__ :is([class^="Dialog__Backdrop-"], [class*=" Dialog__Backdrop-"])', 'click', onButtonClick, {signal});
```

Comment:
What's stopping this from just being
```suggestion
	delegate('[class*="Dialog__Backdrop-"]', 'click', onButtonClick, {signal});
```

Reply:
Used `^=` and `*=` to make sure it wouldn't capture any `...Dialog__Backdrop-...` classes, should be safe to make this change anyways